gensim/__init__.py

```diff
@@ -2,7 +2,7 @@
 of documents.
 """
 
-from gensim import parsing, corpora, matutils, interfaces, models, similarities, summarization, utils  # noqa:F401
+from gensim import parsing, corpora, matutils, interfaces, models, similarities, summarization, utils, similarity_learning  # noqa:F401
```

Comment:
why we need `similarity_learning` submodule? `gensim/models/experimental/...` should be enough

Reply:
Oh
I misunderstood what you meant initially
I thought "models" meant strictly model code, i.e., keras model code

and all other callbacks, preprocessing, etc goes in gensim.similarity_learning

Reply:
no, model should contain all needed code inside :)